composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "broadway/read-model-mongodb",
+    "description": "A MongoDB ReadModel implementation for Broadway.",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Robin van der Vleuten",
+            "email": "robin@webstronauts.co"
+        }
+    ],
+    "homepage" : "https://github.com/broadway/read-model-mongodb",
+    "autoload": {
+        "psr-4": {
+            "Broadway\\ReadModel\\": "src"
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.0.x-dev"
```

Comment:
why not making master the way to the first stable release? Otherwise you'll have to update the alias after each minor release and that's very error prone I know from experience.